packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -376,8 +437,14 @@ Future<T> showModalBottomSheet<T>({
 PersistentBottomSheetController<T> showBottomSheet<T>({
   @required BuildContext context,
   @required WidgetBuilder builder,
+  Color bottomSheetColor,
```

Comment:
For consistency, can this be renamed to `backgroundColor` to match `showModalBottomSheet`